packages/flutter_tools/test/general.shard/features_test.dart

```diff
@@ -78,19 +78,19 @@ void main() {
     }));
 
     test('flutter web help string', () {
-      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter Web on master, dev channels.');
+      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for Web. This setting will take effect on master, dev channels.');
```

Comment:
`"Flutter for Web"` or `"Flutter for web"`?

And, perhaps `"This setting will take effect on the master and dev channels.".